common/history/characters/0_dvg_europe.txt

```diff
@@ -0,0 +1,216 @@
+﻿CHARACTERS = {
```

Comment:
You duplicated characters. This file has been split up. Move the characters you created to a separate file per country. And then delete this one.